src/data/services/EnterpriseAccessApiService.js

```diff
@@ -254,8 +254,11 @@ class EnterpriseAccessApiService {
     return EnterpriseAccessApiService.apiClient().post(url, payload);
   }
 
-  static fetchSubsidyHydratedGroupMembersData(subsidyAccessPolicyUUID, options) {
+  static fetchSubsidyHydratedGroupMembersData(subsidyAccessPolicyUUID, options, selectedEmails) {
     const queryParams = new URLSearchParams(options);
+    if (selectedEmails) {
+      selectedEmails.forEach((email) => queryParams.append('learners', email));
+    }
```

Comment:
[nit/suggestion] no longer need this check if we pass in the learner emails through options